nncf/common/graph/patterns/patterns.py

```diff
@@ -260,3 +258,113 @@ def merge_two_types_of_operations(first_op: Dict, second_op: Dict, label: str) -
         res[GraphPattern.LABEL_ATTR] = label
         return res
     raise RuntimeError('Incorrect dicts of operations')
+
+
+@dataclass
+class PatternDesc:
```

Comment:
Could you please add docstrings for the `PatternDesc` and `PatternNames` classes?